dll/cpl/desk/lang/fr-FR.rc

```diff
@@ -7,26 +7,26 @@ CAPTION "Themes"
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "A theme is a background plus a set of sounds, icons, and other elements to help you personalize your computer with one click.", IDC_STATIC, 5, 5, 235, 30
-    LTEXT "Theme:", IDC_STATIC, 5, 42, 55, 10
+    LTEXT "Un thème est un fond d'écran et un ensemble de sons, icones et autres éléments vous permettant de personnaliser votre ordinateur en un clic.", IDC_STATIC, 5, 5, 235, 30
+    LTEXT "Theme :", IDC_STATIC, 5, 42, 55, 10
```

Comment:
`Thème: ` (the space is after the colon, also better with accents)

Reply:
accent added